shell/pages/home.vue

```diff
@@ -41,17 +41,9 @@ export default {
 
   mixins: [PageHeaderActions],
 
-  async fetch() {
-    const requests = {
-      rancherClusters: this.$store.dispatch('management/findAll', { type: CAPI.RANCHER_CLUSTER }),
-      clusters:        this.$store.dispatch('management/findAll', {
-        type: MANAGEMENT.CLUSTER,
-        opt:  { url: MANAGEMENT.CLUSTER }
-      })
-    };
-    const hash = await allHash(requests);
-
-    this.clusters = hash.rancherClusters;
+  fetch() {
+    setPromiseResult(this.$store.dispatch('management/findAll', { type: CAPI.RANCHER_CLUSTER }), this, 'provClusters', 'Failed to load prov clusters');
```

Comment:
We can't get the output, that would require making the fn `async` and `await`ing the result which would block the flow.  That's the opposite of what the change is trying to do. `setPromiseResult` is a convience  method to avoid doing the same process multiple times in multiple files.

I've split the calls to the function over multiple lines and fixed the `failed to` references